static/src/stylesheets/module/_search-tool.scss

```diff
@@ -44,7 +44,7 @@
     box-sizing: border-box;
     color: $neutral-1;
     padding-top: $gs-baseline / 3;
-    padding-bottom: $gs-baseline;
+    padding-bottom: $gs-baseline / 3;
```

Comment:
George advice was to reduce the padding. It is odly unbalanced in prod currently
<img width="178" alt="screen shot 2016-06-22 at 17 34 33" src="https://cloud.githubusercontent.com/assets/233326/16274912/9b15013e-389f-11e6-8dca-3032f0fbcb07.png">